.github/workflows/ci_docker_build_push.yml

```
@@ -83,3 +85,61 @@ jobs:
+  web:
+    name: Build and push web Docker image
+    runs-on: ubuntu-latest
+    if: github.event.inputs.name || github.event.inputs.new_tag
+    env:
+      IMAGE_NAME: reearth/reearth-visualizer-web
+    defaults:
+      run:
+        working-directory: web
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)